ADDING_NEW_RULES.md

```diff
@@ -0,0 +1,121 @@
+* Check the list of [currently implemented rules](RULES.md) to make sure the rule doesn't already exist.
+* Check the list of [planned future rules](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+rule%22) to see if an issue already exists for the proposed rule.
+    * If no existing issue exists, open a new issue with the ["new rule" label](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+rule%22).
+* Discuss the rule with the community via the Github issue and come to a general consensus on the exact logic, and if it should be an `ERROR` or `WARNING`.  Generally, errors are behavior that directly violate the GTFS documentation.  Warnings are behavior that is not advised (e.g., against best practices) but not explicitly forbidden in the GTSF documentation.
+* Implement new rule using the process below
+
+For the below example, let's look at implementing a new rule that [TODO]
+
+If you want to take a look at a complete set of changes that implement this new rule before diving into the instructions, see [TODO this commit on Github](https://github.com/MobilityData/gtfs-validator).
+
+### 1. Add the rule to [`RULES.md`](RULES.md)
+
+Add the rule to the error or warnings table at the top of [`RULES.md`](RULES.md):
+
+~~~
+
+~~~
+
+...and add a definition of that rule at the bottom of the errors or warnings section:
+
+~~~
+
+~~~
+
+### 2. Add a new `*Notice.java` class for the new rule output
+
+All classes that implements rules output are under the [`notice` package](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice) of the domain layer.
+They must extend either `ErrorNotice` or `WarningNotice` and fit the `*Notice` format.
+
+- Add the error code definition to either `ErrorNotice` or `WarningNotice` class located in the [`base`](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/base) package, for example
+
+`protected static final String E_013 = 13;`
+
+- Add your new class in the [`error`](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error) or [`warning`](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/warning) package accordingly
+
+- override and define an `export` method like the following.
+~~~
+@Override
+public void export(final NoticeExporter exporter) throws IOException {
+    exporter.export(this);
+}
+~~~
+Note: you will have a compiler warning "Cannot resolve method" until you complete step 3
+
+- implement your own constructor, calling `super` and passing it `filename`, `code`, `title`, `description` and `entityId`
+
+Optional: if your Notice has specific data, you have to do the following for each piece of data
```

Comment:
Whenever the itemized list is steps from the above sentence, I'd suggest ending the sentence with a `:`.

For example:

```suggestion
Optional: if your Notice has specific data, you have to do the following for each piece of data:
```